app/src/renderer/vuex/modules/delegates.js

```diff
@@ -17,7 +17,16 @@ export default ({ node }) => {
       Object.assign(delegate, delegate.description)
 
       // TODO: calculate voting power
-      delegate.voting_power = delegate.tokens
+      let divIdx = delegate.tokens.indexOf("/")
```

Comment:
If there is still a '/' in the tokens, we are not converting them correctly in the LCD. :/ let's fix this over there.

Reply:
you mean that this would require us to update the response from the SDK, right ?

Reply:
@faboweb If so I'd merge this and open another issue for that in both the SDK and voyager

Reply:
Let's do that.

Reply:
Can you create the issues?